tchannel/peer_heap.py

```diff
@@ -80,3 +80,12 @@ def peek_peer(self):
             return None
 
         return self.peers[0]
+
+    def remove_peer(self, peer):
+        """Remove the peer from the heap.
+        If the peer's index out of range[0, peer'size), the remove function
+        will do nothing.
```

Comment:
Consider changing the second sentence to,

> Unknown peers are ignored.

Range details irrelevant to API.